nat/src/stateless/compute/mod.rs

```diff
@@ -5,17 +5,23 @@
 
 #![deny(clippy::all, clippy::pedantic)]
```

Comment:
Not a fan of the compute name, I probably would have chosen `natconfig` but not worth holding this up for that either.

Reply:
I wanted to distinguish this from the actual dataplane processing. Not that I like it very much either and it can be easily changed. This PR is on top of https://github.com/githedgehog/dataplane/pull/711, so until we merged that, merging this is worthless.

Reply:
How about something like “build” or “setup” rather than “compute”?